internal/workload/wrapper_test.go

```diff
@@ -55,7 +55,7 @@ var _ = Describe("Workload management", func() {
 			pod := &v1.Pod{ObjectMeta: metav1.ObjectMeta{Name: "pod1"}}
 
 			newPodman.EXPECT().Run(manifestPath, authFilePath).Return([]*podman.PodReport{{Id: "id1"}}, nil)
-			newPodman.EXPECT().GenerateSystemdService(gomock.Any(), gomock.Any()).Return(svc, nil)
+			newPodman.EXPECT().GenerateSystemdService(gomock.Any(), gomock.Any(), gomock.Any()).Return(svc, nil)
```

Comment:
shouldn't we check that at least one manifest path matches here? Mainly because maybe in future changed for some reason and we're not going to notice.